docs/learn/learn-elastic-scaling.md

```diff
@@ -0,0 +1,77 @@
+---
+id: learn-elastic-scaling
+title: Polkadot's Elastic Scaling
+sidebar_label: Elastic Scaling
+description: Pushing Parachains' Boundaries Below 6 seconds Execution Time.
+keywords: [elastic scaling, parachains, coretime, blockspace]
+slug: ../learn-elastic-scaling
+---
+
+The path of parablocks from their creation to their inclusion into the relay chain (discussed in the
+[Parachain Protocol Page](./learn-parachains-protocol.md)) span two domains: the parachain's and
+relay chain's roles. Scaling the Polkadot protocol involves consideration of how parablocks are
+validated, processed, secured, made available for additional checks, and finally included in the
+relay chain as part of its final state
+
+[Asynchronous backing](./learn-async-backing.md) is the first implementation of the parachain
+protocol that alters how parachains produce blocks and how the relay chain processes their data.
+This upgrade will allow parachains to take up to 2 seconds execution time (for producing
+parablocks), while the relay chain will be able to include a parablock every 6 seconds.
+
+With asynchronous backing enabled, parachains can theoretically produce parablocks in the sub-second
+time range. However, to accomplish this, asynchronous backing must temporarily **hold** blocks in
+[unincluded segments](./learn-async-backing.md#unincluded-segments), while parablocks can be
+included into the relay chain every 6 seconds per core. This is the bottleneck that elastic scaling
+addresses. With elastic scaling, parachains can use multiple cores to include multiple parablocks
+within the same relay chain block.
+
+Take, for example, a parachain that wants to submit four parablocks to the relay chain. Without
+elastic scaling, it will take 24 seconds to include all of them through one core. Remember that a
+core is occupied after backing and before inclusion, i.e., for the whole data availability process.
+A block cannot enter a core before the previous block has been declared available.
+
+```
+              R1 <----- R2 <----- R3 <----- R4 <----- R5
+
+C1    |P1     B          I
+      |P2                B         I
+      |P3                          B         I
+      |P4                                    B         I
+
+```
+
+The diagram above shows how the backing and inclusion of parablocks (P) happen within the same relay
+chain block (R). With one core (C1), a parablock is included every 6 seconds. Note how P4 is
+included after 30 seconds (not 24 seconds) because when P1 was pushed to the relay chain for being
+backed, there was no previous parablock.
+
+With elastic scaling, it will take just 12 seconds (3-second block time) to include all four
+parablocks using two cores.
+
+```
+              R1 <----- R2 <----- R3
```

Comment:
This is due to a lack of knowledge on my part - but why would P3 be backed and included before P1?  Although, I know we may not have all the details of how this works yet

We would need to elaborate on ordering and how that works on a basic level.